src/bindings/python/flux/util.py

```diff
@@ -426,6 +426,11 @@ def convert_field(self, value, conv):
             value = super().convert_field(value, conv)
         return value
 
+    def empty_outputs(self):
+        localepoch = datetime.fromtimestamp(0.0).strftime("%FT%T")
+        empty = ("", "0s", "0.0", "0:00:00", "1970-01-01T00:00:00", localepoch)
+        return empty
```

Comment:
This function doesn't seem to use `self` so maybe it can be defined outside of this class. This will allow the unnecessary instantiation of a `UtilFormatter()` object to be dropped in `fliter_empty()` below.